gc_registry/certificate/schemas.py

```diff
@@ -94,6 +94,11 @@ class GranularCertificateBundleBase(BaseModel):
                         standardised energy volume per Granular Certificate, rounded down to the nearest Wh. Equal to
                         (bundle_id_range_end - bundle_id_range_start + 1)."""
     )
+    beneficiary: str | None = Field(
```

Comment:
Are these all the changes from the other PR?

Reply:
Looks like it - didn't see them in time